### PR TITLE
Chatty cyborg

### PR DESCRIPTION
**Pull request by xanderrootslayer**
#### Summary

SUMMARY: Content "Add more lines in speech.json for the Broken Cyborg and Prototype Cyborg"

#### Purpose of change

PURPOSE OF CHANGE: speech.json has lots of lines for the mi-go, but not much for the more obscure Broken Cyborg.

#### Describe the solution

DESCRIBE THE SOLUTION: Throw in 6 to a dozen more lines each for both creatures, plus another Mi-go line because I felt mean. The Broken Cyborg sounds more out of it, while the Prototype Cyborg is more plainly conscious and suffering. Hopefully they will be easier to tell apart at a distance.

#### Describe alternatives you've considered

DESCRIBE ALTERNATIVES YOU'VE CONSIDERED: Less pop culture references. *The Brain that Would Not Die*? Really?

### CODE REVIEW

File: data/json/speech.json [hunk 1]
```diff
@@ -1637,6 +1643,42 @@
...
+  {
+    "type": "speech",
+    "speaker": "mon_broken_cyborg",
+    "sound": "\"I told you to let me die.\"",
+    "volume": 10
+  },
+  {
+    "type": "speech",
+    "speaker": "mon_broken_cyborg",
+    "sound": "\"W-where am I? Why does it hur-REBOOTING IN 59 SECONDS.\"",
```
Comment (ZhilkinSerg) [inline data/json/speech.json]:
```suggestion
    "sound": "\"W-where am I?  Why does it hur-REBOOTING IN 59 SECONDS.\"",
```

File: data/json/speech.json [hunk 2]
```diff
@@ -1637,6 +1643,42 @@
...
+    "volume": 20
+  },
+  {
+    "type": "speech",
+    "speaker": "mon_broken_cyborg",
+    "sound": "\"THIS UNIT IS BROKEN. PLEASE CALL AN ATTENDANT.\"",
```
Comment (ZhilkinSerg) [inline data/json/speech.json]:
```suggestion
    "sound": "\"THIS UNIT IS BROKEN.  PLEASE CALL AN ATTENDANT.\"",
```

File: data/json/speech.json [hunk 3]
```diff
@@ -1655,12 +1697,42 @@
     "sound": "\"I have done what you asked.  Please let me go!\"",
     "volume": 10
   },
+  {
+    "type": "speech",
+    "speaker": "mon_prototype_cyborg",
+    "sound": "\"I cannot... yet I must. How do you calculate that?\"",
```
Comment (ZhilkinSerg) [inline data/json/speech.json]:
```suggestion
    "sound": "\"I cannot... yet I must.  How do you calculate that?\"",
```

File: data/json/speech.json [hunk 4]
```diff
@@ -1655,12 +1697,42 @@
...
+    "volume": 10
+  },
+  {
+    "type": "speech",
+    "speaker": "mon_prototype_cyborg",
+    "sound": "\"This is a dream. This has to be a dream.\"",
```
Comment (ZhilkinSerg) [inline data/json/speech.json]:
```suggestion
    "sound": "\"This is a dream.  This has to be a dream.\"",
```

Comment (xanderrootslayer) [reply data/json/speech.json]: I applied the changes! Thanks for catching those!

**Pull request merged**